optaplanner-benchmarks/pom.xml

```diff
@@ -37,13 +42,41 @@
       <dependency>
         <groupId>org.openjdk.jmh</groupId>
         <artifactId>jmh-core</artifactId>
-        <version>${version.jmh}</version>
+        <version>${jmh.version}</version>
       </dependency>
       <dependency>
         <groupId>org.openjdk.jmh</groupId>
         <artifactId>jmh-generator-annprocess</artifactId>
-        <version>${version.jmh}</version>
+        <version>${jmh.version}</version>
+      </dependency>
+      <!-- Optaplanner -->
+      <dependency>
+        <groupId>org.optaplanner</groupId>
+        <artifactId>optaplanner-bom</artifactId>
+        <version>${version.org.optaplanner}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+      <dependency>
+        <groupId>org.optaplanner</groupId>
+        <artifactId>optaplanner-build-parent</artifactId>
+        <version>${version.org.optaplanner}</version>
+        <type>pom</type>
+        <scope>import</scope>
```

Comment:
There is a dilemma here as well, I can do that, but that means that I would need to provide the version of parent before running mvn install that means that I would need to use the script that changes the version,
I want to use another approach here, so I can pass the version as a parameter. 

Like here: `mvn clean install -DskipTests -Dversion.org.optaplanner=8.1.0.Final`

I still do not see the profit to inherit the whole project setup, could you maybe elaborate what is exactly cons of importing parent instead of inheritance?

Reply:
OK, I have tried it myself and I don't see much benefit in inheriting `optaplanner-build-parent`. And I understand that by importing it we get dependency management for `commons-lang3`, `slf4j-api` etc.

Resolved.